corporate/conftest.py

```diff
@@ -21,3 +31,41 @@ class MockStripeSessionResponse:
 def mock_session(*args, **kwargs):
     """Fixture of MockStripeSessionResponse."""
     return MockStripeSessionResponse()
+
+
+class DottableDict(dict):
+    """A dict that can be accessed by dot notation and by key."""
+
+    def __init__(self, *args, **kwargs):
+        """Initialize the dict."""
+        dict.__init__(self, *args, **kwargs)
+        self.__dict__ = self
+
+    def allowDotting(self, state=True):
+        """Allow dot access."""
+        if state:
+            self.__dict__ = self
+        else:
+            self.__dict__ = dict()
+
+
+class MockStripeCheckoutSession:
+    """Dummy checkout session object."""
+
+    customer = "unique_stripe_id"
+
+    class metadata:
+        """Metadata object."""
+
+        customer_uuid = "asdj"
+
+
+@pytest.fixture
+def create_stripe_checkout_session_event_mock():
```

Comment:
Can you rename this to:
```suggestion
def stripe_checkout_session_event_mock():
```